### PR TITLE
Update quotes.json

### PR DESCRIPTION
**Pull request by farazkhanfk7**
Added another quote by Mahatma Gandhi in this quotes.json file.

### CODE REVIEW

File: quotes.json [hunk 1]
```diff
@@ -3783,6 +3783,10 @@
     "author": "Mahatma Gandhi",
     "text": "We must become the change we want to see."
   },
+  {
+    "author": "Mahatma Gandhi",
+    "text": "Live as if you were to die tomorrow.Learn as if you were to live forever."
```
Comment (nelsonic) [inline quotes.json]: "tomorrow.Learn" >> "tomorrow. Learn" ...
```suggestion
"text": "Live as if you were to die tomorrow. Learn as if you were to live forever."
```
Just adding a space between the full stop and "Learn".

**Comment by nelsonic** [comment]
@farazkhanfk7 we need you to accept the suggested change ⬆️ https://github.com/dwyl/quotes/pull/13#discussion_r362386276

**Comment by farazkhanfk7** [comment]
Thanks for this suggestion and change @nelsonic . 👍 
I came to know about dwyl from a friend. 
And yes this was my first pull request  :) I am a student but I am looking forward to contribute more and more in future.

**Comment by nelsonic** [comment]
@farazkhanfk7 thanks for confirming that your friend introduced you to @dwyl 🎉 

Given that the PR is based a branch that only you have access to, you will need to perform the **`Commit suggestion`** action above ⬆️ https://github.com/dwyl/quotes/pull/13#discussion_r362386276
image

Sample commit message:
```
add space in Gandhi quote between full stop and "Learn"
```
Once that is done the PR will be merged. :octocat: 
Thanks again. 🥇

**Comment by nelsonic** [comment]
P.S. remember to ⭐️ the repository to help _other_ people discover it. 😉

**Comment by farazkhanfk7** [comment]
Thanks @nelsonic for helping me out. 👍

**Comment by iteles** [comment]
Welcome to dwyl @farazkhanfk7, so glad you found us 🚀 🎉

**Pull request merged**